### ISSUE TITLE
Cannot view store pages for individual items after they are bought in a bundle

### ISSUE THREAD

**Issue by erikturn** [itchio/itch.io#1436]
When you buy things in a bundle  (e.g. something with 10 or 20 items like sidescroller or topdown texture collections), you can no longer view the individual store pages for the sub-items....it takes you to the bundle page which doesn't have all the details that the individual pages have, and the bundle page links only to the asset downloads.

You should be able to link back to the bundle page, as well as look at the individual store pages for each sub-item after purchase...

**Comment by leafo** [comment]
I'm not sure I entirely follow what you're describing. When you buy something it doesn't prevent you from viewing the store page anymore. On the download page for a product, there is a link on the top that should take you back to the store page.

**Comment by erikturn** [comment]
Yeah, it wasn't obvious how to find that link and I was getting sent in an
endless circular loop.
I'm able to see the links from the download page to get back to the
original store page now, and it seems now that when I click on items I
already own generally  it's loading the right page and telling me I own
them, so it's a lot better.  A lot of the items that I bought explain
things about the asset on the store page that aren't exactly documented in
the actual download, so you always need to be able to see the original
store page entry....

Thanks!
Erik
